From bc34b4ace42cac34b966cc5fa357452d23c86d5b Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gmail.com>
Date: Fri, 1 Jul 2011 16:04:37 -0400
Subject: [PATCH] backend-cogl: update in line with Cogl API change

cogl_renderer_xlib_set_foreign_display was renamed to
cogl_xlib_renderer_set_foriegn_display so this is the corresponding
change to clutter.

Signed-off-by: Emmanuele Bassi <ebassi@linux.intel.com>

[Alexandre Rostovtsev <tetromino@gmail.com>: backported to clutter-1.7.2]
---
 clutter/egl/clutter-backend-egl.c |    2 +-
 clutter/glx/clutter-backend-glx.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/clutter/egl/clutter-backend-egl.c b/clutter/egl/clutter-backend-egl.c
index 601b4bf..ff1cd2c 100644
--- a/clutter/egl/clutter-backend-egl.c
+++ b/clutter/egl/clutter-backend-egl.c
@@ -300,7 +300,7 @@ clutter_backend_egl_create_context (ClutterBackend  *backend,
 
   backend->cogl_renderer = cogl_renderer_new ();
 #ifdef COGL_HAS_XLIB_SUPPORT
-  cogl_renderer_xlib_set_foreign_display (backend->cogl_renderer,
+  cogl_xlib_renderer_set_foreign_display (backend->cogl_renderer,
                                           backend_x11->xdpy);
 #endif
   if (!cogl_renderer_connect (backend->cogl_renderer, error))
diff --git a/clutter/glx/clutter-backend-glx.c b/clutter/glx/clutter-backend-glx.c
index 76f48ee..03cfb1c 100644
--- a/clutter/glx/clutter-backend-glx.c
+++ b/clutter/glx/clutter-backend-glx.c
@@ -243,7 +243,7 @@ clutter_backend_glx_create_context (ClutterBackend  *backend,
     return TRUE;
 
   backend->cogl_renderer = cogl_renderer_new ();
-  cogl_renderer_xlib_set_foreign_display (backend->cogl_renderer,
+  cogl_xlib_renderer_set_foreign_display (backend->cogl_renderer,
                                           backend_x11->xdpy);
   if (!cogl_renderer_connect (backend->cogl_renderer, error))
     goto error;
-- 
1.7.6

